package/yast2-storage-ng.spec

```diff
@@ -16,7 +16,7 @@
 #
 
 Name:		yast2-storage-ng
-Version:	4.1.15
+Version:	4.1.16
```

Comment:
Please, don't increase version yet. I will do it in my pull request. I don't want to release what is in master right now without some corrections that are on #737

Reply:
ok